frontend-svelte/src/routes/+page.ts

```diff
@@ -0,0 +1,13 @@
+import type { PageLoad } from './$types';
+import { getMe } from '$lib/api/users';
+
+export const ssr = true;
```

Comment:
What's this line for? Does it even make a difference given that `export const ssr = true;` is set in `routes/+layout.ts`?